rosstatic/main.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# TODO: add custom header, footer
+# TODO: symlinks or copy param
+
+import subprocess
+import os
+import shutil
+import rospy
+
+rospy.init_node('rosstatic')
+
+www = os.path.dirname(os.path.realpath(__file__)) + '/www'
+index_file = rospy.get_param('~index', None)
+
+shutil.rmtree(www, ignore_errors=True)  # reset www directory content
```

Comment:
I wonder whether we really need to remove and recreate the whole directory every time.

Reply:
If a package got removed, I don't think its directory should remain to be served.

Reply:
That is true, however individual package symlinks could be removed instead of the whole folder.